src/Microsoft.Owin.Security.Cookies.Shareable/AspNet5TicketSerializer.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.Framework.Internal;
+using Microsoft.Owin.Security.DataHandler.Serializer;
+
+namespace Microsoft.Owin.Security.Cookies.Shareable
+{
+    public class AspNet5TicketSerializer : IDataSerializer<AuthenticationTicket>
```

Comment:
Though it's probably the main target for such a package, the cookies middleware is not the only one using a ticket serializer. Katana's OAuth2 bearer/server middleware also used it massively. Have you considering creating a more generic package for this class, to avoid tying it to cookies?